bots/c/client_lib/src/internal/grpc_bridge.cpp

```
@@ -0,0 +1,539 @@
+
+static t_obj *cache_store(const t_obj &obj)
+{
+	if (g_cache == nullptr)
+	{
+		g_cache_capacity = 64;
+		g_cache = (t_obj **)malloc(sizeof(t_obj *) * g_cache_capacity);
+	}
+	else if (g_cache_count >= g_cache_capacity)
+	{
+		g_cache_capacity *= 2;
+		g_cache = (t_obj **)realloc(g_cache, sizeof(t_obj *) * g_cache_capacity);
```

Cppcheck (2.19.0)

[error] Common realloc mistake

(memleakOnRealloc)